src/test/java/step3/CarTest.java

```diff
@@ -0,0 +1,22 @@
+package step3;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+
+	@DisplayName(value = "조건에 따른 자동차의 전진 또는 정지 테스트")
+	@ParameterizedTest
+	@CsvSource(value = {"3:0", "1:0", "4:1", "6:1", "8:1"}, delimiter = ':')
+	void moveForward(int value, int expected) {
+		Car car = new Car();
+		car.moveForward(value);
+		assertThat(car.getRecord()).isEqualTo(expected);
+	}
```

Comment:
현재 getter를 통해 데이터를 가지고 와서 테스트를 하고 계시네요!
다음과 같이 getter를 사용하지 않고 비교할 수 있습니다. `() -> assertThat(actual).isEqualTo(expect)`
다음과 같이 비교할 수 있도록 리팩토링하시면 좋을 것 같아요!
```suggestion
	void moveForward(int value, int expected) {
		Car expect = new Car();
		expect.moveForward(value);

		Car actual = new Car();
		actual.moveForward(value);

		assertAll(
				() -> assertThat(actual).isEqualTo(expect)
				, () -> assertThat(actual.getRecord()).isEqualTo(expected)
		);
	}
```
![image](https://user-images.githubusercontent.com/20432323/89737827-2b5f0180-daaf-11ea-84a5-9ce387e306cc.png)

힌트는 Car 객체에 두 개의 메서드를 오버라이드 하시면 됩니다!
어떤 메서드를 오버라이드 해야하며 왜 오버라이드하면 다음과 같은 비교가 가능할 수 있는지 
고민해보아요 :)